java/ql/test/library-tests/dataflow/records/A.java

```diff
@@ -0,0 +1,28 @@
+public class A {
+  record Pair(Object x, Object y) { }
```

Comment:
Could you also add some test with fields that have explicit "get" methods? I think one field with a sensible get method that just returns the field (has flow) and one silly get method that returns a constant (no flow)

Reply:
Done.